src/olympia/devhub/views.py

```diff
@@ -1373,10 +1357,14 @@ def version_list(request, addon_id, addon):
     new_file_form = forms.NewVersionForm(None, addon=addon, request=request)
     is_admin = acl.action_allowed(request, 'ReviewerAdminTools', 'View')
 
+    payload = jwt_api_settings.JWT_PAYLOAD_HANDLER(request.user)
+    token = jwt_api_settings.JWT_ENCODE_HANDLER(payload)
```

Comment:
Instead of generating a new token, can you re-use the one that should exist in the cookies ? (It may not exist in the tests by default)